app/src/main/java/com/example/drawingactivity/ColorUtils.kt

```diff
@@ -0,0 +1,74 @@
+package com.example.drawingactivity
+
+import android.graphics.Color
+import java.lang.Integer.parseInt
+
+object ColorUtils {
+
+    /**
+     * A data class that contains color values.
+     * Contains: Hue, Saturation, Value, Red, Green, Blue, and Hex Value.
+     */
+    data class HsvState(
```

Comment:
This isn't really a state. Just `Hsv` should do

Reply:
renamed